src/pages/components/paragraph.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Paragraph
+
+description: Use Paragraph to render text blocks with the appropriate Garden styling.
```

Comment:
Let's change to: 

Use Paragraph to render blocks of text with Garden styling.